src/status_im/ui/screens/discover/events.cljs

```diff
@@ -27,26 +29,30 @@
                                       [message-id discover]))
                                (into {})))))
 
+;; todo(goranjovic): at the moment we do nothing when a status without hashtags is posted
+;; but we probably should post a special "delete" status that removes any previous
+;; hashtag statuses in that scenario
```

Comment:
That sound important. Can you create an issue for it and reference it here?

Reply:
It's still being scoped. The purpose of the comment was to explain why the check for hashtags is done by this event and not by the caller. Hm, I guess I could have said that explicitly, so good point!